src/components/Display.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Display({ result }) {
+  return (
+    <div>
+      <h1>{result}</h1>
+    </div>
+  );
+}
```

Comment:
- [ ] As it said in the requirements, we need to set a default value for the result props. This way, passing the props it's not going to be required. The default value should be a string.
```JS
Display.defaultProps = {
  result: '0',
};

Display.propTypes = {
  result: PropTypes.string,
};
```
![Screenshot from 2021-04-08 09-01-10](https://user-images.githubusercontent.com/30977845/114023242-03e05480-9849-11eb-9bce-20f85e099d8c.png)
![Screenshot from 2021-04-08 09-01-22](https://user-images.githubusercontent.com/30977845/114023254-0773db80-9849-11eb-80ce-9e45562ea9df.png)